code42.json

```diff
@@ -629,6 +629,331 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "get alert details",
+            "identifier": "get_alert_details",
+            "description": "Get alert details",
+            "verbose": "",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "alert_id": {
+                    "description": "Alert ID",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "alert"
+                    ],
+                    "value_list": [],
+                    "default": "",
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.alert_id",
+                    "data_type": "string",
+                    "contains": [
+                        "alert"
```

Comment:
I don't this in this list of built-in contains types. Are we able to use custom ones out of the box like this?

Reply:
Oops, that was a mistake. Removed